specification/adhybridhealthservice/resource-manager/Microsoft.ADHybridHealthService/stable/2014-01-01/ADHybridHealthService.json

```diff
@@ -3535,6 +3535,125 @@
           }
         }
       }
+    },
+    "/providers/Microsoft.ADHybridHealthService/services/{serviceName}/ipAddressAggregates": {
+      "get": {
+        "tags": [
+          "Reports"
+        ],
+        "description": "Gets the ipAddressAggregates for a given service.",
+        "x-ms-examples": {
+          "list_ipAddressAggregatesByService": {
+            "$ref": "./examples/IpAddressAggregates.json"
+          }
+        },
+        "operationId": "list_ipAddressAggregatesByService",
+        "parameters": [
+          {
+            "name": "serviceName",
+            "in": "path",
+            "description": "The name of the service.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "skiptoken",
+            "in": "query",
+            "description": "A continuationtoken value returned in paginated result to load different pages.",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The list of IP addresses.",
+            "schema": {
+              "$ref": "#/definitions/IpAddressAggregates"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/providers/Microsoft.ADHybridHealthService/services/{serviceName}/ipAddressAggregateSettings": {
+      "get": {
+        "tags": [
+          "Reports"
+        ],
+        "description": "Gets the ipAddressAggregateSettings.",
+        "x-ms-examples": {
+          "get_ipAddressAggregateSettings": {
+            "$ref": "./examples/IpAddressAggregateSettings.json"
+          }
+        },
+        "operationId": "get_ipAddressAggregateSettings",
```

Comment:
Let's use List_ instead of Get_ since it is on the collection resource. Note operation id typically will be turned into SDK, PSH and CLI commands thus the scrutiny here.

Reply:
ahh, ok

Reply:
Fixed